test/HaxelibTests.hx

```diff
@@ -20,38 +19,37 @@ class HaxelibTests {
 	}
 
 	static function cmdSucceed(cmd:String, ?args:Array<String>):Bool {
-		var p = try {
+		final p = try {
 			new Process(cmd, args);
 		} catch(e:Dynamic) {
 			return false;
 		}
-		var exitCode = p.exitCode();
+		final exitCode = p.exitCode();
 		p.close();
 		return exitCode == 0;
 	}
 
 	static public function deleteDirectory(dir:String) {
 		if (!FileSystem.exists(dir)) return;
-		var exitCode = switch (Sys.systemName()) {
+		final exitCode = switch (Sys.systemName()) {
 			case "Windows":
 				Sys.command("rmdir", ["/S", "/Q", StringTools.replace(FileSystem.fullPath(dir), "/", "\\")]);
 			case _:
 				Sys.command("rm", ["-rf", dir]);
 		}
-		if (exitCode != 0) {
+		if (exitCode != 0)
```

Comment:
Personally, I don't like this brace-less style. But if this what the rest of haxelib does then I agree that we should make it consistent.

Reply:
I had a look through and it does seem like the braces are preferred. Might be a good idea to setup a haxe checkstyle file for stuff like this at some point.